openmc/plots.py

```diff
@@ -1489,8 +1667,14 @@ def from_xml_element(cls, elem):
             plot_type = e.get('type')
             if plot_type == 'projection':
                 plots.append(ProjectionPlot.from_xml_element(e))
-            else:
+            elif plot_type == 'phong':
+                plots.append(PhongPlot.from_xml_element(e))
+            elif plot_type == 'voxel':
                 plots.append(Plot.from_xml_element(e))
+            elif plot_type == 'slice':
```

Comment:
While I think this is an improvement, does it break backward compatibility? I can't recall how long the `Plot.type` attribute has been around...

Reply:
Pretty sure it does not, no.